webapp/src/containers/LiquidityPage/components/LiquidityInfo/index.tsx

```diff
@@ -40,8 +40,7 @@ const LiquidityInfo: React.FunctionComponent<LiquidityInfoProps> = (
   }, []);
 
   const liquidityAmount = (percentage, reserve) => {
-    const liquidityAmount = (Number(percentage) / 100) * reserve;
-    return liquidityAmount.toFixed(8).toString() || '0';
+    return new BigNumber(percentage || 0).div(100).times(reserve).toFixed(8);
```

Comment:
when deal with ratio or percentage, if possible (like this 1 liner)
always multiply first then divide

eg
1 / 3 * 3 toFixed(8) = 0.99999999
1 * / 3 toFixed(8) = 1.00000000

Reply:
Hi @ivan-zynesis, I think it should be ok with BigNumber. I've tested using BigNumber and Number, both of them has the same result. 
![image](https://user-images.githubusercontent.com/25013382/105790822-f47a9a80-5fbf-11eb-9a86-cc0c4dbde51a.png)